### DISCUSSION TITLE
571A9F42-1F1F-4FF9-ABE5-58D843621579

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#145] [General]
# HD 102365

HD 102365, also known as Gliese 442, is a nearby binary star system consisting of a Sun-like star and a red dwarf only about 30.31 light-years away in the constellation Centaurus. The larger star of the two has a mini-neptunian planet that has an orbit comparable to Mercury's distance to the Sun

https://celestia.mobi/addon?item=571A9F42-1F1F-4FF9-ABE5-58D843621579

**Comment by exocritic5457** [comment]
No uncited Simpsons reference, good addon.

**Comment by TyphoonRagasa** [comment]
I think this add-on should be moved into Hypothetical & Disproven
https://arxiv.org/abs/2507.07514
